app/docs/how-to/streamline-curbside.md

```diff
@@ -0,0 +1,255 @@
+
+#### Finding your Trello API Key and Token
+
+You can find your **Trello API Key** [here](https://trello.com/app-key) under the **Key** section.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/trello-api-key.png" width="75%">
+</p>
+
+Once you have recorded the key, click on the **Token** link to generate your **Trello API Token**.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/trello-api-token-allow.png" width="75%">
+</p>
+
+Scroll to the bottom of the page, click **Allow** and you should be able to see and record your **API Token**.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/trello-api-token.png" width="75%">
+</p>
+
+#### Finding a ListId of your *Waiting Curbside Customers* list
+
+To find this, perform the following steps:
+
+* Open the board that contains the list you wish to use. In the address bar, you will see a URL like `https://trello.com/b/DRzgczUF/customer-waitlist`.
+  
+* Change the value in your address bar to `https://trello.com/b/DRzgczUF.json?fields=id,name&lists=open` using **your unique** board identifier. 
+* This URL will display all open lists on your Trello board. Choose the one you want and capture the `id` value. In this example, we're using the `Requests` list with an id of `5eb4674aed5ff17eae53760f`.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/trello-list-id.png" width="75%">
+</p>
+
+::: warning The board ID is unique to you!
+* The `DRzgczUF` value used in this example is a **unique** board identifier and the value for your Trello board **will be different**.
+:::
+
+
+### Putting Routegy and Trello together
+
+The final step is to connect Routegy and Trello using a [touch handler](/reference/touch-handlers/) that will create a new Trello card every time someone interacts with your touchpoint. 
+
+#### Step 1: Create a touch handler
+
+Go back to your Routegy workspace, select **Handlers** tab, and click **Create handler** button.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/handlers-tab.png" width="90%">
+</p>
+
+Give your new handler a name and description, and choose **Trello** from the **Handler Type** dropdown. Next, enter your **Trello API Key**, **API Token**, and **ListId** that you captured when configuring the Trello board [earlier](#finding-your-trello-api-key-and-token). 
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/handler-trello.png" width="75%">
+</p>
+
+Finally, scroll down to the **Card name** field, and replace its value with the following template: 
+```
+{{ name }} is waiting in {{ car }}, ordered via {{ order_type }}, arrived at {{ context.touch.created_at | date:"g:iA" }}
+```
+**Card name** field controls how Routegy will name Trello cards that it creates. The template above customizes the name so it will contain a complete information about arriving customers (e.g. *Pawel is waiting in Red Honda, ordered via Online, arrived at 5:42PM*)
```

Comment:
1) Start sentence with "The"

"contain a complete information". Remove the "a" and "complete".